perfkitbenchmarker/providers/aws/aws_virtual_machine.py

```diff
@@ -545,7 +545,8 @@ class RhelBasedAwsVirtualMachine(AwsVirtualMachine,
 
   def __init__(self, vm_spec):
     super(RhelBasedAwsVirtualMachine, self).__init__(vm_spec)
-    self.user_name = 'ec2-user'
+    user_name_set = FLAGS['aws_user_name'].present
```

Comment:
Let me know if the temp variable is okay, without it the line is longer than 80 characters and it looks weird to split a ternary operator over multiple lines.